### ISSUE TITLE
Domestic Politics

### ISSUE THREAD

**Issue by Drunk-Monk** [We-the-People-civ4col-mod/Mod#318]
There is a real lack of internal politics and "friction" from declaring independence. There are no entrenched interests, or defenders, or loyalists of the old regime. Historically all the independence struggles were destructive civil wars, however in the game all the colonies constantly desire independence with no possibility of support, falling or changing. Its trivial to get 100% support, and your only penalty is having difficulty with trading with Europe, at which point you have enough money / resources to win.

From watching lets plays of the Original Colonisation, it seems like you would face opposition from your own colonists if you didn’t have 100% support, and the King would encourage Indian tribes to turn against you. I would like to see something similar introduced in We The People, to shake up the late game. The Constitution is also relatively useless, as it doesn’t even interact with the unique elements of the mod like slavery. In general the game lacks a compelling internal dynamic, tending just to focus on production as outlined here #317

**Comment by Nightinggale** [comment]
The constitution is set as civics. Civics can now provide CivEffects. The design goal of CivEffects is that they can provide whatever feature we can get from civics, traits, founding fathers etc. as well as whatever new features we add, which applies to just a single player (or each player in a team in case of FF).

This means we have the "engine" for making civics a whole lot more interesting. The question is how we should use it. It can be anything from free promotions, (un)locking units or whatever else we can come up with. Anything gained from techs in BTS would also be valid to add to CivEffects.

We could also consider adding more civic groups, some for the declaration and some for... well always. This way we can make pre-independence make use of the civic system to grant the player some ability to make the games more diverse through domestic policies.

Other ideas for increased domestic policy control would be welcome.

**Comment by Drunk-Monk** [comment]
Adding civilisation style civics, would be really interesting in the long term. I think for now, making the Constitution have relevance to the game, including all the new features added through the mod, should receive first priority, as its one of the most poorly implemented features in the game. My suggestions for a new Constitution:

**All Men Are Free**

At the moment, this is very poorly implemented. It gives every settlement two free indentured servants, regardless of population which is very arbitrary. They often end up deserting, so usually prove to be an annoyance rather than a benefit. Instead we should update this to reflect the new features introduced in the mod, by making this actually free all enslaved units, as a trade off it would prevent the player from acquiring new slaves.

**Slavery**

At the moment, this gives a 50% bonus to all raw yields. I think that's a little too much, but the idea is fine.

**Controlled Arms**

At the moment, this gives a 50% bonus to Great General Emergence, which is fine.

**Right to Bear Arms** 

At the moment, this gives a 2 strength per colonist. I will check this out but it seems badly implemented. Would probably be better it was replaced with a promotion or experience bonus of some kind.

**Elections**

At the moment, this gives a 50% bonus to Liberty Bells which seems pretty pointless as you have already won by this point. Or do other people declare independence before they are at 100%? I think some form of happiness bonus would be a good substitute. 

**Monarchy**

At the moment, this allows you to continue trading with Europe, which is kind of pointless as a few turns later 30+ Men of War turn up, and you should already have enough money and resources. I think this should boost your relations with other European kings and their colonies, so they are more likely to provide you with aid. 

**Theocracy**

At the moment, this converts Crosses into Hammers, again a little pointless as there is very little left to build. I think this should boost your relations with the Archbishop, so he is more likely to provide you with aid. 

**Separation of Church and State**

At the moment, this converts Crosses into Liberty Bells, again this seems pretty pointless as you have already won by this point. I'm not sure what new dynamic could be added here, as religion in general is not very well implemented.

**Manifest Destiny**

At the moment, this gives a 50% bonus against Natives and dominates their borders. I think the bonus is too strong and as referenced here: #297 the border issue is not properly implemented, but if it was this would be ok.

**Native rights**

At the moment, this strengthens relations with Natives and makes auto peace. I have already referenced why this is poorly implemented: #298 but its not a bad idea in principle.

**Comment by Nightinggale** [comment]
> free all enslaved units, as a trade off it would prevent the player from acquiring new slaves.

I was thinking something like that today as well. CivEffects already has the ability to ban units. If we add convert unit A to B, then we will have code support for this in a generic approach, which can be reused for other purposes. Since each Era can grant a CivEffect, adding a conversion ability means we can make progressing to a certain era result in converting unit A to unit B. I don't know what we should use that ability for, but planning ahead and gaining that ability "for free" can't be wrong.

> replaced with a promotion 

We have the code already to grant a free promotion to all units of a certain player. It can optionally be limited to certain professions or unitcombat, but all combat units is also an option. I prefer the promotion approach because it is the simplest from a programming point of view and in this context, simple means fast execution.

> religion in general is not very well implemented.

Ray mentioned he came up with the name "Religion and Revolution" because he had plans for a greatly expanded religion system. However he never implemented it and he didn't give details for what he had in mind. This means whatever he was thinking is lost by now, but naturally that won't stop us from coming up with our own system if we can figure out something, which will improve the game.

-----

I have no comments to the other civics right now other than I tend to agree with what you wrote.

**Comment by Drunk-Monk** [comment]
> Ray mentioned he came up with the name "Religion and Revolution" because he had plans for a greatly expanded religion system. However he never implemented it and he didn't give details for what he had in mind. This means whatever he was thinking is lost by now, but naturally that won't stop us from coming up with our own system if we can figure out something, which will improve the game.
> 

Yes, I was aware and I agree, but I think that we need to handle things carefully to stop things becoming overly gamey, and to bear in mind that religious affiliation and its politics was not black and white. Catholic France and Spain were bitter enemies for much of the period, as were Protestant Sweden and Denmark and majority Protestant England and the Netherlands. France and England both had monarchs of a minority religion, or who were sympathetic to the minority. Most of the colonists of the 13 Colonies were Protestants, but disagreed vehemently on the 'correct' way to worship.

I think a series of event to simulate the reformation would be good. I also think adding a simple bonus/malus to diplomatic relationships to those who follow the same different faith would be ok. Also allowing the Indians to convert to a faith, once that faith has a certain amount of followers like in Civ would be good.

> 
> Other ideas for increased domestic policy control would be welcome.

I have been questioning how we could simulate loyalty or revolutionary fervour, its pretty difficult, as historians are still arguing as to why people chose to side with each faction in the American revolution, most citizens were neutral. For many Latin American countries, there was little support for revolution and the 'Liberators' were actually invaders. 

The feature that I am wondering if it could be added, is if colonists, (or maybe colonies?) could have a level of loyalty to you or the King that is hidden, until you have a certain amount of support for revolution. At which point you would face riots or perhaps revolts in the colonies that opposed you, necessitating the player to garrison troops there, or make some form of decision to maintain the peace (or not).

I think that, having the King donate Horses and Weapons to encourage hostile tribes to attack you, like in the original colonisation, would not be to difficult to implement.   

I would really welcome other players to contribute to their ideas to this thread, or to explain how other games (Paradox plaza's?) manage this mechanic.
